### PR TITLE
INHEART-409: Create predefined categories

### PR DESCRIPTION
**Pull request by Pomarancz88**

### CODE REVIEW

**Review by captain-yossarian — APPROVED** [review]

**Review by pmusialik-intive — APPROVED** [review]

**Review by pawelroguski — COMMENTED** [review]
I don't know that this solution will work. To save auction we need relation in database between auction and category. In this case we have only mocked categories.

**Review by pawelroguski — COMMENTED** [review]
I left video in comment in task. Something not work properly

**Review by pmusialik-intive — COMMENTED** [review]
Ok that's right, we need those keys in DB, hmmm

File: package.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "license": "MIT",
   "scripts": {
     "start": "nx run-many --target=serve --projects=backend,mobile",
+    "start:backend": "nx run-many --target=serve --projects=backend",
```
Comment (pawelroguski) [inline package.json:7]: This change should be in other commit ;)

**Pull request closed without merging**